buff.lic

```diff
@@ -21,6 +21,10 @@ class Waggle
 
     waggle_spells = @settings.waggle_sets
 
+    if args.spells['prehunt_buffs']
+      return unless waggle_spells[args.spells]
+    end
+
```

Comment:
Due to us not using a flag for buffing or not buffing, and buffing if the prehunt_buffs exists, I added the logic you see here. Let me know suggestions if ruby handles this better or if my logic could just be all around better.